vagrant/values.yaml

```diff
@@ -29,3 +29,31 @@ sumologic:
 telegraf-operator:
   enabled: true
   replicaCount: 1
+
+falco:
+  enabled: true
+  customRules:
+    # Mark the following as known k8s api callers:
+    # * fluentd and its plugins
+    # * grafana sidecar
+    # * terraform provider started in setup job
+    # * telegraf operator
+    # * kube state metrics server
+    # * prometheus
+    # * prometheus operator
+    rules_user_known_k8s_api_callers.yaml: |-
+      - macro: user_known_contact_k8s_api_server_activities
+        condition: >
+          (proc.pcmdline = "fluentd /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins") or
+          (proc.cmdline = "fluentd /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins") or
+          (proc.cmdline = "ruby -Eascii-8bit:ascii-8bit /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins --under-supervisor") or
+          (proc.cmdline = "event_loop -Eascii-8bit:ascii-8bit /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins --under-supervisor") or
+          (proc.cmdline = "watch_endpoints -Eascii-8bit:ascii-8bit /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins --under-supervisor") or
+          (proc.cmdline = "watch_events -Eascii-8bit:ascii-8bit /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins --under-supervisor") or
+          (proc.cmdline = "filter_kuberne* /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins") or
```

Comment:
Does it make sense to use some regex here? Is it possible?

Reply:
All I can find regarding regexes in falcos docs is that you can disable sets of rules matching regexes so I guess not

Reply:
ok, so `filter_kuberne*` is actually process name?

Reply:
This is exactly verbatim what was in the falco log, copy-pasted.